data/shaders/skybox_vs.glsl

```diff
@@ -0,0 +1,21 @@
+#version 150 core
+#include globals
+
+out vec3 v_TexCoord;
+
+void main() {
+    vec2 pos = gl_VertexID == 0 ? vec2(-1.0, -1.0) :
+               gl_VertexID == 1 ? vec2(-1.0,  1.0) :
+               gl_VertexID == 3 ? vec2( 1.0,  1.0) :
+                                  vec2( 1.0, -1.0) ;
+
+    vec4 a_Position = vec4(pos.xy, 1.0, 1.0);
+
+    mat4 inverseProj = inverse(u_Proj);
```

Comment:
FYI, usually, matrix inverse operations are performed on the CPU. Since there are a tiny amount of vertices in the case of a skybox it doesn't really matter in this case.

Reply:
It can be faster than on CPU due to only 4 vertices.

Reply:
Please don't do `inverse` in the shader. You added more stuff to `Globals` anyway, so better make it ready for computation on the CPU side. Inversing a matrix IIRC is not even available on non-GLSL shaders.